src/models/predict_model.py

```diff
@@ -149,13 +154,15 @@ def main(args, save_path="", evaluating_test=True):  # noqa: C901
 
     # Convert two sets into two branch with one set in both and one set in only one (Use for this file)
     test = test_2
+    test_sv = test_3
     params = params_2
     params_sv = params_3
 
     vv_branch = args.vv_branch
+    set_onnx = args.set_onnx
 
     prediction = np.array([])
-    batch_size = 1024
+    batch_size = 1000  # 1024
```

Comment:
maybe this should be a command line argument with a default of 1024 (or 2048 or whatever)?